src/test/java/io/confluent/examples/streams/ApplicationResetIntegrationTest.java

```diff
@@ -16,8 +16,9 @@
 package io.confluent.examples.streams;
 
 import io.confluent.examples.streams.kafka.EmbeddedSingleNodeKafkaCluster;
-import kafka.admin.AdminClient;
```

Comment:
We still used the old admin client? Was about time to change this. Nice catch!

Reply:
Thanks! I was also surprised.